capstone/target/portfolio-1/index.html

```diff
@@ -2,14 +2,14 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
+    <title>AppName</title>
+    <link rel="stylesheet" href="styles/style.css">
```

Comment:
Try to proactively scope your .css into targeted files, using a shared style.css file for only styles which are shared across more pages, maybe even only which are shared on ALL pages.